sympy/core/relational.py

```diff
@@ -1105,7 +1118,11 @@ def is_le(lhs, rhs):
 
 
 def is_ge(lhs, rhs):
+    if type(lhs) == Basic or type(rhs) == Basic:
+        raise TypeError("Cannnot compare inequality on Basic")
```

Comment:
This should probably be
```
    if not (isinstance(lhs, Expr) and isinstance(rhs, Expr)):
        raise TypeError("Can only compare inequalities with Expr")
```
It's very unusual to have an object whose type is literally Basic.